mmdet/models/data_preprocessors/data_preprocessor.py

```diff
@@ -58,6 +59,8 @@ class DetDataPreprocessor(ImgDataPreprocessor):
             Defaults to False.
         rgb_to_bgr (bool): whether to convert image from RGB to RGB.
             Defaults to False.
+        with_box_wrapped (bool): Whether to keep the ``BaseBoxes`` wrapper of
```

Comment:
boxlist2tensor might be more clear, indicating whether to convert box list to tensor